doc/man3/SSL_CTX_set_early_cb.pod

```diff
@@ -53,6 +54,14 @@ from the ClientHello on a per-extension basis.  For the provided wire
 protocol extension type value, the extension value and length are returned
 in the output parameters (if present).
 
+SSL_early_get1_extensions_present() can be used prior to SSL_early_get0_ext(),
+to determine which extensions are present in the ClientHello before querying
+for them.  The B<out> and B<outlen> parameters are both required, and the
+caller must free B<*out> on success.  The contents of B<*out> is an array
+of network-byte-order 16-bit integers of TLS extension types, in the order
+they appear in the ClientHello.  B<outlen> contains the size, in bytes, of
```

Comment:
maybe just return an array of ints in local-host format, which are the extensions seen.  That seems simpler.